egs/vaevc/vcc2018v1/conf/slurm.conf

```diff
@@ -0,0 +1,9 @@
+command sbatch --export=PATH  --ntasks-per-node=1
```

Comment:
Is it ok to as symbolic link? I did not try.

Reply:
Same as above.